CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0 (2015-09-11)
+- Support all URL parameters when using `Client.#scroll`
+- BREAKING: Moved some `Scroller` reader methods into `Scroller.opts`
```

Comment:
This is very nitpicky, but I tend to put breaking changes at the top of the changes list.